src/components/policy/policy_external/include/policy/policy_table/types.h

```diff
@@ -485,7 +485,7 @@ struct ConsumerFriendlyMessages : CompositeType {
 
 struct ModuleMeta : CompositeType {
  public:
-  Optional<String<0, 250> > ccpu_version;
+  Optional<String<0, 500> > ccpu_version;
```

Comment:
Why do we need this? Isn't that enough 250 chars for ccpu version?

Reply:
@YarikMamykin according to the API